README.md

```diff
@@ -31,18 +31,19 @@ Usage:
   spanner-cli [OPTIONS]
 
 spanner:
-  -p, --project=    (required) GCP Project ID. [$SPANNER_PROJECT_ID]
-  -i, --instance=   (required) Cloud Spanner Instance ID [$SPANNER_INSTANCE_ID]
-  -d, --database=   (required) Cloud Spanner Database ID. [$SPANNER_DATABASE_ID]
-  -e, --execute=    Execute SQL statement and quit.
-  -f, --file=       Execute SQL statement from file and quit.
-  -t, --table       Display output in table format for batch mode.
-  -v, --verbose     Display verbose output.
-      --credential= Use the specific credential file
-      --prompt=     Set the prompt to the specified format
-      --history=    Set the history file to the specified path
-      --priority=   Set default request priority (HIGH|MEDIUM|LOW)
-      --role=       Use the specific database role
+  -p, --project=       (required) GCP Project ID. [$SPANNER_PROJECT_ID]
+  -i, --instance=      (required) Cloud Spanner Instance ID [$SPANNER_INSTANCE_ID]
+  -d, --database=      (required) Cloud Spanner Database ID. [$SPANNER_DATABASE_ID]
+  -e, --execute=       Execute SQL statement and quit.
+  -f, --file=          Execute SQL statement from file and quit.
+  -t, --table          Display output in table format for batch mode.
+  -v, --verbose        Display verbose output.
+      --credential=    Use the specific credential file
+      --prompt=        Set the prompt to the specified format
+      --history=       Set the history file to the specified path
+      --priority=      Set default request priority (HIGH|MEDIUM|LOW)
+      --role=          Use the specific database role
+      --directed-read= Set the location and replica type for directed reads.
```

Comment:
Can we add the details to this help message as well?

Reply:
added the same message as help message.